01-Login/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gradle:4.2-jdk8
+
+WORKDIR /tmp
+ADD . /tmp
+
+RUN gradle build
+
+CMD java -jar build/libs/auth0-spring-security-mvc.war
```

Comment:
We [are telling users](https://auth0.com/docs/quickstart/webapp/java-spring-security-mvc/01-login#run-the-sample) to run the sample using `bootRun` as that's the task that spring boot adds to the project. What's the benefit on doing it this other way?